src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    margin: 0 3px;
+    margin-bottom: 40px;
+  }
+
+  &__info-name {
+    font-size: 12px;
+    font-weight: $font-medium;
```

Comment:
you set this value as a default for your html tag. 
I recommend you to find that styles that used the most and apply them to html. 
And for other selectors add only at that places that you have a difference